src/index.js

```diff
@@ -31,12 +31,20 @@ const App = () => {
  const [index, setIndex] = useState(0);
  const [redirectOption, setredirectOption] = useState();
  const [noteTracker, setnoteTracker] = useState(null);
-
+ const highlights = []
+ const preHighlight = JSON.parse(localStorage.getItem("myNoteDb"))
+ if(preHighlight)
+ {
+preHighlight.map((data,index) => {
+    highlights.push(new Date(data.id))
+})
+ }
  // Calender Configs see: @natscale/react-calender
  const [calOptions] = useState({
   useDarkMode: false,
   startOfWeek: 0,
   disablePast: true,
```

Comment:
We could make it work by setting this as false (disablePast flag) or removing that line, but then the visual UX of past days in the widget would be gone..

Reply:
That that should work fine 😀

Reply:
Also forgot to mention that there's currently no way to delete a note. And the freshly highlighted date won't be seen on the widget until the page is refreshed

Reply:
I'll soon add those an make another pr 😜